desktop/app/console-helper.js

```diff
@@ -16,7 +16,7 @@ let fileWritable = null
 // If the file is too big, let's reset the log
 if (logFile) {
   fs.access(logFile, fs.W_OK, err => {
-    if (err && err.errno !== -2 /* file does not exist */) {
+    if (err && (err.errno !== -2 && err.errno !== -4058) /* file does not exist */) {
```

Comment:
This check should also be platform specific, but it's fine for now as a quick fix for windows